### ISSUE TITLE
“Save search result” button missing tip-text or aria-label.

### ISSUE THREAD

**Issue by DavidIsaacBPS** [HHS/simpler-grants-gov#6801]
### Summary

Given page is Search Results
When I hover over the star next to a search result,
Then I expected to see or hear some text saying what the star does
But instead there is nothing.

Note: this is a minor usability/accessibility issue.  Found by ASTA.

### Reproducibility

I tried multiple times and saw the bug every time

### How to reproduce the bug

See summary.

### Browser

Chrome

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct
